src/styles/blocks/catalog.scss

```diff
@@ -2,24 +2,32 @@
   display: flex;
   flex-wrap: wrap;
 
-  max-width: 944px;
+  max-width: $catalog-width;
   margin: 0 auto;
-  padding: 50px 0;
+  padding: $catalog-padding-y 0;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color $trans-duration $smooth,
+      transform $trans-duration $smooth;
+
     &:not(:nth-child(4n)) {
-      margin-right: 48px;
+      margin-right: $space-btwn-cards;
```

Comment:
not the best naming option either. I would say you don't need this at all, we usually don't create variables for margins. There's atomic CSS libs for that, so you could just add classname `mr-48` and it will apply the needed margin. But for now I would say it is better to stick to usual margin property

Reply:
ok, I'll consider it in the future.
As for now, I can't drop it to usual margin property, as this variable is used in the catalog width calculation.
![image](https://user-images.githubusercontent.com/127268424/235627387-f004ef5f-2816-4bca-9f42-2e122dab3a5a.png)